ruma-events/src/room/message.rs

```diff
@@ -690,3 +759,235 @@ pub struct CustomEventContent {
     #[serde(flatten)]
     pub data: JsonObject,
 }
+
+fn get_plain_quote_fallback(original_message: &MessageEvent) -> String {
+    match &original_message.content.msgtype {
+        MessageType::Audio(_) => {
+            format!("> <{:?}> sent an audio file.", original_message.sender)
+        }
+        MessageType::Emote(content) => {
+            format!("> * <{:?}> {}", original_message.sender, content.body)
+        }
+        MessageType::File(_) => {
+            format!("> <{:?}> sent a file.", original_message.sender)
+        }
+        MessageType::Image(_) => {
+            format!("> <{:?}> sent an image.", original_message.sender)
+        }
+        MessageType::Location(content) => {
+            format!("> <{:?}> {}", original_message.sender, content.body)
+        }
+        MessageType::Notice(content) => {
+            format!("> <{:?}> {}", original_message.sender, content.body)
+        }
+        MessageType::ServerNotice(content) => {
+            format!("> <{:?}> {}", original_message.sender, content.body)
+        }
+        MessageType::Text(content) => {
+            format!("> <{:?}> {}", original_message.sender, content.body)
+        }
+        MessageType::Video(_) => {
+            format!("> <{:?}> sent a video.", original_message.sender)
+        }
+        MessageType::_Custom(content) => {
+            format!(
+                "> <{:?}> {}",
+                original_message.sender,
+                content.data["body"].as_str().unwrap_or("")
+            )
+        }
+        #[cfg(feature = "unstable-pre-spec")]
+        MessageType::VerificationRequest(content) => {
+            format!("> <{:?}> {}", original_message.sender, content.body)
+        }
+    }
+}
+
+fn get_html_quote_fallback(original_message: &MessageEvent) -> String {
+    match &original_message.content.msgtype {
+        MessageType::Audio(_) => {
+            formatdoc! {"
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        sent an audio file.
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+            }
```

Comment:
I know the indoc docs use curly braces but I'd much prefer this to use parentheses like usually with `format!` too.

```suggestion
            formatdoc!(
                "
                <mx-reply>
                    <blockquote>
                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
                        <br />
                        sent an audio file.
                    </blockquote>
                </mx-reply>
                ",
                room_id = original_message.room_id,
                event_id = original_message.event_id,
                sender = original_message.sender,
            )
```

Reply:
Same below.

Reply:
Changed to parentheses.